workspaces/util/src/__tests__/chunkBySize-test.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, expect, it } from "@jest/globals";
```

Comment:
Thanks so much for the PR @ismailnyza -- I appreciate your willingness to change things quickly and hopefully it still feels somewhat simple 😄 

I think this will be ready to merge after the latest round of comments.

Also, just to confirm, are you ok with the MIT License for your contributions?

Reply:
hi i cant seem to stop the comma from being removed after saved here

expect([...result]).toStrictEqual([[1, ], [3]]);  intended 

expect([...result]).toStrictEqual([[1], [3]]); // Sparse array after save

Reply:
A trailing comma doesn't actually make it a sparse array! [3, 1, 4, ] is an array of length 3, with 3 defined elements, the same as [3, 1, 4]. That's why the formatter is willing to make this change.

I think a sparse array with the missing elements in the middle should be sufficient. If you want the sparse part to be at the end you could do something like:

```ts
const sparseArray = Array(10);
sparseArray[1] = ...;
sparseArray[3] = ...;
```